Dockerfile

```diff
@@ -68,91 +70,91 @@ RUN /usr/local/bin/install-plugins.sh       \
   ace-editor:1.1                 \
   ansicolor:0.5.2                \
   antisamy-markup-formatter:1.5  \
-  artifactory:2.12.2             \
+  artifactory:2.13.1             \
   authentication-tokens:1.3      \
-  azure-credentials:1.2          \
-  azure-vm-agents:0.4.7.1        \
-  branch-api:2.0.11              \
+  azure-credentials:1.3.1        \
+  azure-vm-agents:0.5.0          \
+  branch-api:2.0.16              \
   build-name-setter:1.6.7        \
-  build-timeout:1.18             \
-  cloudbees-folder:6.1.2         \
+  build-timeout:1.19             \
+  cloudbees-folder:6.2.1         \
   conditional-buildstep:1.3.6    \
-  config-file-provider:2.16.3    \
-  copyartifact:1.38.1            \
+  config-file-provider:2.16.4    \
+  copyartifact:1.39              \
   cvs:2.13                       \
   docker-build-publish:1.3.2     \
-  docker-workflow:1.13           \
-  durable-task:1.14              \
-  ec2:1.36                       \
+  docker-workflow:1.14           \
+  durable-task:1.17              \
+  ec2:1.38                       \
   embeddable-build-status:1.9    \
   external-monitor-job:1.7       \
   ghprb:1.39.0                   \
-  git:3.5.1                      \
-  git-client:2.5.0               \
+  git:3.6.4                      \
+  git-client:2.6.0               \
   git-server:1.7                 \
-  github:1.28.0                  \
-  github-api:1.86                \
-  github-branch-source:2.2.3     \
+  github:1.28.1                  \
+  github-api:1.90                \
+  github-branch-source:2.3.1     \
   github-organization-folder:1.6 \
-  gitlab:1.4.8                   \
-  gradle:1.27.1                  \
+  gitlab:1.5.2                   \
+  gradle:1.28                    \
   greenballs:1.15                \
   handlebars:1.1.1               \
-  ivy:1.27.1                     \
-  jackson2-api:2.7.3             \
-  job-dsl:1.65                   \
-  jobConfigHistory:2.17          \
-  jquery:1.11.2-1                \
-  ldap:1.17                      \
+  ivy:1.28                       \
+  jackson2-api:2.8.10.1          \
+  job-dsl:1.66                   \
+  jobConfigHistory:2.18          \
+  jquery:1.12.4-0                \
+  ldap:1.18                      \
   mapdb-api:1.0.9.0              \
   marathon:1.6.0                 \
-  matrix-auth:1.7                \
-  matrix-project:1.11            \
-  maven-plugin:2.17              \
+  matrix-auth:2.2                \
```

Comment:
1 -> 2; anything to worry about here?

Reply:
https://wiki.jenkins.io/display/JENKINS/Matrix+Authorization+Strategy+Plugin
Under "Important upgrade notes":

> This release uses a new on-disk format for permissions inheritance options. Existing options will be retained when upgrading, but downgrading to older versions may result in failures to load job or folder permission data, or different (typically additional) permissions being granted after the downgrade.